tests/unit-tests/test_evaluator.py

```diff
@@ -375,6 +375,145 @@ def check_evaluate_infer(mock_compute_score):
 
     th.distributed.destroy_process_group()
 
+def test_classification_evaluator():
```

Comment:
Is this almost same as test_acc_evaluator?

Reply:
Yes, it is very similar. Will remove it after all replacements implemented.